FlowersOrder/FlowersOrderScreen/FlowersOrderTableViewCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TableViewCell.swift
+//  FlowersOrder
+//
+//  Created by Dragos Cosma on 03/02/2020.
+//
+
+import UIKit
+
+class FlowersOrderTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var smallImage: UIImageView!
+    @IBOutlet weak var priceLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    
+    var viewModel: FlowerCellViewModelProtocol? {
+        didSet {
+            setupCell()
```

Comment:
I think we need to also clear the labels and imageView in "prepareForReuse", otherwise when the cells are reused and the ViewModel has some nil properties, those rows will contain the old label values that were reused.